client/js/components/statement/assessmentTable/DpExportModal.vue

```diff
@@ -45,25 +45,25 @@
             role="tabpanel">
             <fieldset
               v-if="options.pdf.anonymize || options.pdf.obscure"
-              class="u-mb-0_5 u-pb-0_5">
+              class="mb-2 pb-2">
```

Comment:
Here and below: The `mb-2` is not applied, since the fieldset has `margin-bottom: 0 !important`. `u-mb-0_5` worked because it also has the `!important` declaration.

(Before:
![spacing_before](https://github.com/user-attachments/assets/8030d027-5707-42ad-a8b2-c836b72ea5ef)
After:
![spacing_after](https://github.com/user-attachments/assets/f2a9aa44-9d19-4130-9b8b-2b6da749f3fb))

Reply:
Thanks, I changed it here [e8844ac](https://github.com/demos-europe/demosplan-core/pull/3582/commits/e8844ac2723cb99ae252471c8e55d791c4fe4c89)